tensornetwork/backends/pytorch/pytorch_backend.py

```diff
@@ -475,3 +475,17 @@ def sign(self, tensor: Tensor) -> Tensor:
   
   def item(self, tensor):
     return tensor.item()
+
+  def power(self, a: Tensor, b: Tensor) -> Tensor:
+    """
+    Returns the power of tensor a to the value of b.
```

Comment:
the wording of this docstring seems somewhat imprecise. Can you fix this?

Reply:
>  """
>     Returns the exponentiation of tensor a raised to b.
>     In the case b is a tensor, then the power is by element
>       with a as the base and b as the exponent.

>     Args:
>       a: The tensor that contains the base.
>       b: The tensor that contains a single scalar.
>     """

This is the change I made to it, once we resolve the other request I will push the changes